src/main/java/uk/gov/hmcts/reform/finrem/caseorchestration/helper/DocumentHelper.java

```diff
@@ -381,4 +382,10 @@ public static String getRespondentFullNameContested(CaseDetails caseDetails) {
     public List<HearingOrderCollectionData> getFinalOrderDocuments(Map<String, Object> caseData) {
         return objectMapper.convertValue(caseData.get(FINAL_ORDER_COLLECTION), new TypeReference<>() {});
     }
+
+    public  List<HearingOrderCollectionData> getHearingOrderDocuments(Map<String, Object> caseData) {
```

Comment:
does this method exist privately elsewhere? if so can we remove it from there and use this one?

Reply:
It does. Sorted now.